README.md

```diff
@@ -1,29 +1,50 @@
 
-## Overview
+During training, AMIDES' machine learning models for both the misuse classification and rule attribution components are trained using a set of SIEM detection rules and historical benign events taken from an organization's corporate network.
 
 ![amides_architecture](./docs/amides.png)
 
-AMIDES is trained using a set of SIEM detection rules and historical benign events taken from an organization's corporate network.
-During operation, incoming events are passed to the rule matching component and the feature extraction component, which transforms the events into feature vectors. The features required for vectorization have been learned during the training phase. The feature vectors are then passed to the misuse classification component, which classifies events as malicious or benign. In case of a malicious result, the feature vector is passed to the rule attribution component, which generates a ranked list of SIEM rules potentially evaded by the event. In the final step, potential alerts of the rule matching and both machine learning components are merged into a single alert by the alert generation component.
+During operation, incoming events are passed to the rule matching component and the feature extraction component, which transforms the events into feature vectors. The features used for vectorization have been learned during the training phase. The feature vectors are then passed to the misuse classification component, which classifies events as malicious or benign. In case of a malicious result, the feature vector is passed to the rule attribution component, which generates a ranked list of SIEM rules potentially evaded by the event. In the final step, potential alerts of the rule matching and both machine learning components are merged into a single alert by the alert generation component.
 
 ## System Requirements
 
-AMIDES was developed and tested on Linux using Python 3.10. Before attempting to use `amides`, make sure you have
+AMIDES was developed and tested on Linux using Python 3.10. It can be run by either installing it (onto your local system/locally) or using the provided docker quickstart environment. Before attempting to install and run `amides`, make sure you have
 
-- Physical or virtual host with a Linux-based OS
+- A physical or virtual host with a Linux-based OS
 - A minimum of 8 GB of RAM
 - At least 2 GB of HDD space
-- Python 3.10 (or newer)
-- jq
 
-The repository contains a `Dockerfile` that creates a quickstart environment for the `amides` package. For testing purposes, we highly recommend to use the quickstart environment. Building and using the environment has been tested with `docker 20.10`.
+### Local Installation
```

Comment:
Since we recommend the quickstart environment, maybe switch these two sections (i.e., quickstart first)?

Reply:
That fits together better then.